src/ring_theory/norm.lean

```diff
@@ -160,4 +161,81 @@ algebra.norm_eq_zero_iff_of_basis (basis.of_vector_space K L)
 
 end eq_zero_iff
 
+namespace intermediate_field.adjoin_simple
+
+open intermediate_field
+
+variable (K)
+
+lemma norm_eq_norm_adjoin [finite_dimensional K L] [is_separable K L] (x : L) :
+  norm K x = (norm K (adjoin_simple.gen K x)) ^ (finrank K⟮x⟯ L) :=
```

Comment:
I think it's clear enough without the extra parentheses, but feel free to keep them.
```suggestion
  norm K x = norm K (adjoin_simple.gen K x) ^ finrank K⟮x⟯ L :=
```